.travis.yml

```diff
@@ -1,13 +1,12 @@
 sudo: required
 dist: trusty
-language: cpp
+language: python
```

Comment:
I don't see a reason to change this, we are doing a C++ library build after all.

Reply:
Sorry, I'm being lazy so didn't get the exact reason, but without this changing Travis CI doesn't start matrix build for
```yaml
python:
  - "2.7"
  - "3.4"
```

Reply:
Ah, ok, that makes sense.